integration/spark-common-cluster-test/src/test/scala/org/apache/spark/sql/common/util/QueryTest.scala

```diff
@@ -45,6 +45,7 @@ class QueryTest extends PlanTest with Suite {
   Locale.setDefault(Locale.US)
   CarbonProperties.getInstance()
     .addProperty(CarbonCommonConstants.VALIDATE_DIRECT_QUERY_ON_DATAMAP, "false")
+    .addProperty(CarbonCommonConstants.CARBON_READER_SUPPORT, "false");
```

Comment:
All these test (DataLoadingTestCase , QueryTest) are not related to SDK flow.  Why they are impacted?
If its becasue of the flag is not cleared, I think its better to use ThreadLocal to store sdk flag and construct the SDKReader instead of CARBON_READER_SUPPORT parameter.

Reply:
I don't know why  it show SDV fail. It shouldn't clean all. 

ok, I changed.